sem_covid/services/secret_manager.py

```diff
@@ -7,8 +7,10 @@
 
 def get_vault_secret(secret_key: str, default_value: str = None):
     from sem_covid import config
-    vault_client = VaultAdapter(config.VAULT_ADDR, config.VAULT_TOKEN, SECRET_MOUNT)
-    secrets_dict = vault_client.load_secrets_for_paths(SECRET_PATHS)
+    vault_client = VaultSecretsStore(config.VAULT_ADDR, config.VAULT_TOKEN, SECRET_MOUNT)
+    secrets_dict = {}
+    for path in SECRET_PATHS:
+        secrets_dict.update(vault_client.get_secrets(path))
```

Comment:
Now I realise that the logic of getting content from a single "secret path" or from a list of secrets, could be implemented in the vault_client (where the parameter could be either a string or a list); but this is not that important here.

Reply:
It is good that the logic of getting the concrete secrets specific to the project is in the service layer, where the project use cases are implemented.